gulpfile.js

```diff
@@ -135,7 +135,7 @@ const PRODUCTION_HEADER = [
 const buildDist = function(filename, opts, isProduction) {
   const webpackOpts = {
     debug: !isProduction,
-    externals: [/^[-/a-zA-Z0-9]+$/],
+    externals: [/^[-a-z0-9]+(\/.+)?$/],
```

Comment:
What is this intending to match in words? It is pretty close to just `/^[-a-z0-9]/`. If that's the intention, it'd be good to simplify to that. If not, it might be wrong.